noetic/distribution.yaml

```diff
@@ -9915,8 +9915,8 @@ repositories:
     release:
       tags:
         release: release/noetic/{package}/{version}
-      url: https://github.com/nobleo/rplidar_ros-release.git
-      version: 2.0.0-1
+      url: https://github.com/Slamtec/rplidar_ros-release.git
```

Comment:
Hm, this is changing the release repository that we've been using up until now.  @Timple, are you OK with this?